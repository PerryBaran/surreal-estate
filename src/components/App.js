import React from "react";
import style from "../styles/app.module.css";

const App = () => {
  return (
    <div className={style.app}>
      <h1>Surreal Estate</h1>
    </div>
  );
}

export default App;
