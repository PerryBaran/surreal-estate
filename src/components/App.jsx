import React from "react";
import { Routes, Route } from "react-router-dom";
import style from "../styles/app.module.css";
import NavBar from "./NavBar";
import Properties from "./Properties";
import AddProperty from "./AddProperty";

const App = () => {
  return (
    <div className={style.app}>
      <NavBar />
      <Routes>
        <Route path="/" element={<Properties />} />
        <Route path="add-property" element={<AddProperty />} />
      </Routes>
    </div>
  );
};

export default App;
