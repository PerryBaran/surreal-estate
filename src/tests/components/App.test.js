import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import { render } from "@testing-library/react";
import App from "../../components/App";

test("renders learn react link", () => {
  const { asFragment } = render(
    <Router>
      <App />
    </Router>
  );

  expect(asFragment()).toMatchSnapshot();
});
