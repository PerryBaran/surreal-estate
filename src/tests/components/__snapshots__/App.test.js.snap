// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<DocumentFragment>
  <div
    class="app"
  >
    <nav
      class="navbar"
    >
      <img
        alt="Surreal Estate"
        class="logo"
        src="logo.png"
      />
      <ul
        class="links"
      >
        <li
          class="item"
        >
          <a
            href="/"
          >
            View Properties
          </a>
        </li>
        <li
          class="item"
        >
          <a
            href="/add-property"
          >
            Add a Property
          </a>
        </li>
      </ul>
    </nav>
    <div
      class="properties"
    >
      <div
        class="properties-array"
      />
    </div>
  </div>
</DocumentFragment>
`;
